django_netjsonconfig/base/admin.py

```diff
@@ -362,6 +365,62 @@ class AbstractTemplateAdmin(BaseConfigAdmin):
     class Media(BaseConfigAdmin.Media):
         js = BaseConfigAdmin.Media.js + [static('{0}js/{1}'.format(prefix, 'template_admin.js'))]
 
+    def delete_view(self, request, object_id, extra_context=None):
+        """
+        send unsubscription notifications to template designer
+        when deleted from the detail page.
+        """
+        if request.POST:
+            template = self.model.objects.get(pk=object_id)
+            subscriber_url = '{0}://{1}'.format(request.META.get('wsgi.url_scheme'),
+                                                request.get_host())
+            subscribe.delay(template.id, template.url, subscriber_url, subscribe=False)
```

Comment:
why can't this operation be triggered from the model?

What if we implement the same feature in an API?